robotidy/transformers/OrderSettings.py

```diff
@@ -142,23 +143,30 @@ def order_settings(node, setting_types, before, after):
         if not node.body:
             return node
         settings = dict()
-        rest = []
+        rest_before = []
```

Comment:
These variable names do not help to understand what value they store. Maybe some comment or name change?

Reply:
When I was adding comments to resolve your request I've noticed this code could be written better - so instead of comment there is now shorter easier to understand code + better names :)

Reply:
Much better! 😉